qiskit_experiments/library/randomized_benchmarking/rb_experiment.py

```diff
@@ -242,4 +255,27 @@ def _metadata(self):
         for run_opt in ["meas_level", "meas_return"]:
             if hasattr(self.run_options, run_opt):
                 metadata[run_opt] = getattr(self.run_options, run_opt)
+
+        # Set constraints to compute basis gates EPGs from an estimated EPC
+        gate_error_ratio = self.experiment_options.gate_error_ratio
```

Comment:
I think this should be a different option name to the analysis. If you want to use a concrete gate error ratio dict you should set the analysis option value directly, not the experiment option value which just gets moved to the analysis option value later.

The experiment option value should be something like `default_gate_error_ratio = True/False`. Then you can add a helper method to this class wrapping the below logic like:

```
def _default_gate_error_ratio(self):
    if not self.experiment_options.default_gate_error_ratio:
        return None

   # calculate and return the dict as you do below
``` 

Then you could either store this in metadata as you do below, or set the corresponding analysis option in the `_finalize` method.

Reply:
I realized that we don't need to compute the error ratio in experiments. This can be always reconstructed from gate counts in circuit metadata.